packages/nodejs/src/clientV2.ts

```diff
@@ -0,0 +1,167 @@
+import { ChannelCredentials, Metadata } from '@grpc/grpc-js';
+import {
+    AccountAddress as AccountAddressLocal,
+    CredentialRegistrationId as CredentialRegistrationIdLocal,
+} from '@concordium/common-sdk';
+import {
+    AccountAddress,
+    BlockHashInput,
+    CryptographicParameters,
+    Empty,
+    NextAccountSequenceNumber,
+    AccountInfoRequest,
+    AccountIdentifierInput,
+    AccountInfo
+} from "./grpc/v2/concordium/types"
+import { QueriesClient } from "./grpc/v2/concordium/service.client"
+import {GrpcTransport} from "@protobuf-ts/grpc-transport";
+
+export type AccountIdentifierInputLocal = AccountAddressLocal | CredentialRegistrationIdLocal | bigint
+
+//////////////////////
+// Helper Functions //
+//////////////////////
+
+function getBlockHashInput(blockHash?: Uint8Array): BlockHashInput {
+    let blockHashInput: any = {};
+
+    if (blockHash) {
+        blockHashInput = {
+            oneofKind: "given",
+            given: {value: blockHash}
+        };
+    } else {
+        blockHashInput = {
+            oneofKind: "lastFinal",
+            lastFinal: Empty
+        };
+    }
+
+    return {blockHashInput : blockHashInput}
+}
+
+function getAccountIdentifierInput(accountIdentifier: AccountIdentifierInputLocal): AccountIdentifierInput {
+    let returnIdentifier: any = {};
+
+    if (accountIdentifier instanceof AccountAddressLocal) {
+        const decodedAddress = new Uint8Array(accountIdentifier.decodedAddress);
+        returnIdentifier.oneofKind = "address"
+        returnIdentifier.address = {value: decodedAddress}
+    } else if (accountIdentifier instanceof CredentialRegistrationIdLocal) {
```

Comment:
I'd prefer we don't use `instanceof` for this.
(This will be used for the web-sdk as well, and there we had an issue with version mismatch causing the check to fail, you can see that I've changed in v1)